src/Contracts/ABIs/DAOFactory.json

```diff
@@ -0,0 +1,367 @@
+[
```

Comment:
let's not copy the ABIs here, but get them from the "migration" dependency as in here https://github.com/daostack/common-scripts/blob/master/settings.js#L7
In this way, they'll be much easier to upgrade when we get a new version